### PR TITLE
attempt to load middleware listed in the config that may be in node m…

### PR DESCRIPTION
**Pull request by ryshep111**
…odules

Tested by listing a node module in the middleware config. That module was loaded properly. Anything that was already loaded in the bootstrapping process will be unaffected.

### CODE REVIEW

**Comment by ryshep111** [comment]
If the service loader cannot find the middleware, an error will be thrown and the server start will fail. This does not differ from the previous behavior because an error would have been thrown by initConsumers() if it got to that point.

**Comment by seanpk** [comment]
Looks good, but let's also add a test case to protect us from future regression.

File: test/integration/testLoader.js [hunk 1]
```diff
@@ -48,4 +49,34 @@ describe('SERVER10 - handler with invalid name should fail on startup', function
...
+
+    it('Launch server and check for failure', function (done) {
+        util.launch('server10', function(output) {
+            assert.ok(output.indexOf('Names cannot contain periods') > -1);
+            done();
+        });
+    });
+});
+
+describe.only('SERVER11 - middleware should get loaded from node modules', function () {
```
Comment (seanpk) [inline test/integration/testLoader.js]: should the `.only()` be removed?

Comment (ryshep111) [inline test/integration/testLoader.js]: yes, good catch.

**Pull request merged**